feat(msk-alpha): MSK Kafka versions 2.8.2.tiered and 3.5.1 and StorageMode property (#27560)
aws/aws-cdk: packages/@aws-cdk/aws-msk-alpha/README.md

}
```

## Storage Mode

> AWS MSK supports the use of [tiered storage](https://docs.aws.amazon.com/msk/latest/developerguide/msk-tiered-storage.html) on Kafka versions suffixed with `.tiered`. On compatiable Kafka versions, the storage mode of the cluster needs to be set to `TIERED` to enable the feature.

Reviewer: ```suggestion
AWS MSK supports the use of [tiered storage](https://docs.aws.amazon.com/msk/latest/developerguide/msk-tiered-storage.html) on Kafka versions suffixed with `.tiered`. 
On compatible Kafka versions, the storage mode of the cluster needs to be set to `StorageMode.TIERED` to enable the feature.

To change the storage mode of a Kafka cluster, use the `storageMode` property:
```

```ts
declare const vpc: ec2.Vpc;
declare const bucket: s3.IBucket;

> const cluster = new msk.Cluster(this, 'cluster', {

Reviewer: ```suggestion

const cluster = new msk.Cluster(this, 'cluster', {
```

}
```

## Storage Mode

> AWS MSK supports the use of [tiered storage](https://docs.aws.amazon.com/msk/latest/developerguide/msk-tiered-storage.html) on Kafka versions suffixed with `.tiered`. 
> On compatible Kafka versions, the storage mode of the cluster needs to be set to `StorageMode.TIERED` to enable the feature.
> 
> To change the storage mode of a Kafka cluster, use the `storageMode` property:

Reviewer: ```suggestion
You can configure an MSK cluster storage mode using the `storageMode` property.

Tiered storage is a low-cost storage tier for Amazon MSK that scales to virtually unlimited storage,
making it cost-effective to build streaming data applications.

> Visit [Tiered storage](https://docs.aws.amazon.com/msk/latest/developerguide/msk-tiered-storage.html) for more details.

```
A further revision to make it clearer.

}
```

## Storage Mode

You can configure an MSK cluster storage mode using the `storageMode` property.

Tiered storage is a low-cost storage tier for Amazon MSK that scales to virtually unlimited storage,
making it cost-effective to build streaming data applications.

> Visit [Tiered storage](https://docs.aws.amazon.com/msk/latest/developerguide/msk-tiered-storage.html) for more details.

> 

Reviewer 2: ```suggestion
```

```ts
declare const vpc: ec2.Vpc;
declare const bucket: s3.IBucket;

const cluster = new msk.Cluster(this, 'cluster', {
  clusterName: 'myCluster',
  kafkaVersion: msk.KafkaVersion.V2_8_2_TIERED,
  vpc,

>   storageMode: msk.StorageMode.TIERED

Reviewer 2: ```suggestion
  storageMode: msk.StorageMode.TIERED,
```
